### PR TITLE
Feature/Add Comment & Like Button

### PR DESCRIPTION
**Pull request by yigitm**
In this PR:

- Comment form is added to the user post path
- Like button is added to the user post path

Thank you for your time.

### CODE REVIEW

File: app/views/posts/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  
+  <div>
+    <%= f.label :text %><br>
+    <%= f.text_area :text %>
+  </div>
+
+  <div>
+    <%= f.submit "Create post" %><br>
+  </div>
+<% end %>
```
Comment (ahmetbozaci) [inline app/views/posts/new.html.erb:15]: - [ ] To create a post ı have to type this URL `http://127.0.0.1:3000/users/1/posts/new`. Please add a link that opens `create post from` on the post page

File: app/controllers/comments_controller.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      render user_post_path(current_user.id, @post.id), alert: 'Comments not created!'
+    end
+  end
+
+  private
+
+  def comment_params
+    params.require(:comment).permit(:text)
+  end
+end
```
Comment (ahmetbozaci) [inline app/controllers/comments_controller.rb:20]: - [ ] Please add a new method in the comment controller as you did in the post controller.

image

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# STATUS: Changes Required  ♻️


 @ ✋🏼 

#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Nice to have 💯
___


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._
-  This error appears after ı click `see all post button`. Please fix it. It is happening on the second user page. The button on the first user page is working well.
image

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ]  These requirements should be added to the `view milestone`. Please add it.

image
- [ ] This user's area should be clickable. To see the user's post ı have to write the route in the address bar.

- [ ] Posts containers are should be clickable also. This should be added in the last milestone. Please fix it.
image


___

### Cheers and Happy coding! 💻 👏

almost

___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

File: app/views/posts/new.html.erb (hunk 1 above)
Comment (yigitm) [reply app/views/posts/new.html.erb:15]: Thank you. Link is added.

File: app/controllers/comments_controller.rb (hunk 2 above)
Comment (yigitm) [reply app/controllers/comments_controller.rb:20]: Thank you. New action with comment instance added.

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

To Highlight: 
- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashboard. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**